Makefile

```diff
@@ -37,6 +37,7 @@ test: keys wasm ## Run the tests
 	$(MAKE) -C ./node $@
 	$(MAKE) -C ./wallet-core $@
 	$(MAKE) -C ./rusk/ $@
+	$(MAKE) -C ./rusk-wallet/ $@
```

Comment:
Would it make sense to add rusk-wallet build as part of `make rusk` or introduce it as a separate `make rusk-wallet` command like we do for the web-wallet?